crates/modelardb_compression/src/models/swing.rs

```diff
@@ -56,6 +56,10 @@ pub struct Swing {
     /// Intercept for the linear function specifying the lower bound for the
     /// current model.
     lower_bound_intercept: f64,
+    /// Computation of the slope to minimize MSE for the current model 
+    /// and holds the numenator and denominator of Equation 6 
+    /// described in Section 3.2 of [Swing and Slide paper].
```

Comment:
I am not sure this link works and actually links to the paper.  `[Swing and Slide paper](https://dl.acm.org/doi/10.14778/1687627.1687645)` can be used to link to the paper.

Reply:
Fixed. Removed the link since it is given above.